### PR TITLE
Add Pulsar Community Weekly Update - 2020-07-25 ~ 2020-07-31 

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2020/2020-07/2020-07-31-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+- [Webinar]
+
+  - 07/28: StreamNative Webinar - "How to Operate Pulsar in Production" by Matteo Merli, Joe Frances, Addison Higham, and Sijie Guo
+
+    Recording: https://www.youtube.com/watch?v=mncXc_T6JkU
+
+### Blog / Article
+
+- Pulsar vs. Kafka: A More Accurate Perspective from Features and Performance to Use Cases and Community
```
Comment (Jennifer88huang) [inline content/2020/2020-07/2020-07-31-pulsar-weekly.md:105]:
```suggestion
- Pulsar vs. Kafka: A More Accurate Perspective from Use Cases and Community to Features and Performance 
```

Comment (Jennifer88huang) [reply content/2020/2020-07/2020-07-31-pulsar-weekly.md:105]: Please update the name in last weekly update.  Thank you.

**Comment by Jennifer88huang** [comment]
It seems that we miss this blog last week https://kesque.com/running-hundreds-of-ml-jobs-in-parallel-with-ease-how-whatify-simplified-its-ml-pipeline-using-kesque/. It's published on July 22.

**Comment by Huanli-Meng** [comment]
> It seems that we miss this blog last week https://kesque.com/running-hundreds-of-ml-jobs-in-parallel-with-ease-how-whatify-simplified-its-ml-pipeline-using-kesque/. It's published on July 22.

it is comes from our competitor. should we cover it in our weekly report?

File: content/2020/2020-07/2020-07-31-pulsar-weekly.md (hunk 1 above)
Comment (Huanli-Meng) [reply content/2020/2020-07/2020-07-31-pulsar-weekly.md:105]: ok

**Comment by Jennifer88huang** [comment]
> > It seems that we miss this blog last week https://kesque.com/running-hundreds-of-ml-jobs-in-parallel-with-ease-how-whatify-simplified-its-ml-pipeline-using-kesque/. It's published on July 22.
> 
> it is comes from our competitor. should we cover it in our weekly report?

Let's discuss it tomorrow.

**Pull request merged**